set.mm

```diff
@@ -52394,23 +52438,34 @@ necessary if all involved classes exist as sets (i.e. are not proper
   ${
     $d ph w $.  $d A w x $.  $d A y $.  $d w y z $.  $d x z $.
     $( Move indexed union inside an ordered-pair class abstraction.
-       (Contributed by Stefan O'Rear, 20-Feb-2015.) $)
+       (Contributed by Stefan O'Rear, 20-Feb-2015.)  Avoid ~ ax-sep ,
+       ~ ax-nul , ~ ax-pr .  (Revised by SN, 11-Nov-2024.) $)
     iunopab $p |- U_ z e. A { <. x , y >. | ph } =
         { <. x , y >. | E. z e. A ph } $=
+      ( vw cv copab wcel wrex cab cop wceq wa wex ciun wb rexcom4 exbii bitri
+      cvv elopabw elv rexbii r19.42v abbii df-iun df-opab 3eqtr4i ) FGZABCHZIZD
+      EJZFKUJBGCGLMZADEJZNZCOZBOZFKDEUKPUOBCHUMURFUMUNANZCOZBOZDEJZURULVADEULVA
+      QFABCUJUAUBUCUDVBUTDEJZBOURUTDBERVCUQBVCUSDEJZCOUQUSDCERVDUPCUNADEUESTSTT
+      UFDFEUKUGUOBCFUHUI $.
+
+    $( Obsolete version of ~ iunopab as of 11-Dec-2024.  (Contributed by Stefan
+       O'Rear, 20-Feb-2015.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    iunopabOLD $p |- U_ z e. A { <. x , y >. | ph } =
+        { <. x , y >. | E. z e. A ph } $=
       ( vw cv copab wcel wrex cab cop wceq wex ciun elopab rexcom4 exbii bitri
       wa rexbii r19.42v abbii df-iun df-opab 3eqtr4i ) FGZABCHZIZDEJZFKUGBGCGLM
       ZADEJZTZCNZBNZFKDEUHOULBCHUJUOFUJUKATZCNZBNZDEJZUOUIURDEABCUGPUAUSUQDEJZB
       NUOUQDBEQUTUNBUTUPDEJZCNUNUPDCEQVAUMCUKADEUBRSRSSUCDFEUHUDULBCFUEUF $.
   $}
 
   ${
-    $d A x y $.  $d R x y $.
+    $d R x $.  $d R y $.
     $( Membership in an ordered-pair class abstraction defined by a binary
-       relation.  (Contributed by AV, 16-Feb-2021.) $)
+       relation.  (Contributed by AV, 16-Feb-2021.)  (Proof shortened by SN,
+       11-Dec-2024.) $)
     elopabr $p |- ( A e. { <. x , y >. | x R y } -> A e. R ) $=
-      ( cv wbr copab wcel cop wceq wa wex elopab df-br biimpi eleq1 syl5ibr imp
-      exlimivv sylbi ) CAEZBEZDFZABGHCUAUBIZJZUCKZBLALCDHZUCABCMUFUGABUEUCUGUCU
-      GUEUDDHZUCUHUAUBDNOCUDDPQRST $.
+      ( cv wbr copab opabss sseli ) AEBEDFABGDCABDHI $.
```

Comment:
This not only shortens the proof, but reduces dependencies on axioms as well, and removes disjointness conditions.  Perhaps use a revision tag instead.

Reply:
I think "Proof shortened" is sufficient.

Reply:
To be honest I was debating not making a tag/old version at all since the proof is automatically found with `improve all /3`

Reply:
Thanks for being honest in this question.  As a possible solution here we have tags like (Proof shortened by OpenAI, 25-Mar-2020.).